components/SectionHeaderEdit/index.tsx

```diff
@@ -12,25 +12,65 @@ interface SectionHeaderEditProps {
 }
 
 const SectionHeaderEdit: React.FC<SectionHeaderEditProps> = ({
-                                                           title,
-                                                           sectionNumber,
-                                                           editUrl,
-                                                           onMoveUp,
-                                                           onMoveDown
-                                                         }) => {
+                                                               title,
+                                                               sectionNumber,
+                                                               editUrl,
+                                                               onMoveUp,
+                                                               onMoveDown
+                                                             }) => {
+
+  const UpArrowIcon = () => (
+    <svg
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M18 15L12 9L6 15"
+        stroke="currentColor"
+        strokeWidth="2"
+        strokeLinecap="round"
+        strokeLinejoin="round"
+      />
+    </svg>
+  );
+
+  const DownArrowIcon = () => (
+    <svg
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M6 9L12 15L18 9"
+        stroke="currentColor"
+        strokeWidth="2"
+        strokeLinecap="round"
+        strokeLinejoin="round"
+      />
+    </svg>
+  );
+
+
```

Comment:
L60: The <div> needs a `role=listitem` since it's contained within a `role=list` div.